network-api/networkapi/wagtailpages/templates/wagtailpages/homepage.html

```diff
@@ -31,172 +31,33 @@ <h1 class="h1-heading">{{ page.hero_headline }}</h1>
-      <div class="row">
-        <div class="col-12 mb-5">
-          <h2>{{ page.partner_heading }}</h2>
-          {% if page.partner_intro_text %}
-            <p>
-              {{ page.partner_intro_text }}
-            </p>
-          {% endif %}
-          <a href="{{ page.partner_page.url }}">
-            {{ page.partner_page_text }}
-          </a>
-        </div>
-        {% for orderable in page.partner_logos.all %}
-          {% image orderable.logo width-200 as logo_img %}
-          <div class="col-2">
-            {% if orderable.link %}
-              <a href="{{ orderable.link }}">
-                <img src="{{ logo_img.url }}" alt="{{ logo_img.alt }}">
-              </a>
-            {% else %}
-              <img src="{{ logo_img.url }}" alt="{{ logo_img.alt }}">
-            {% endif %}
-          </div>
-        {% endfor %}
-      </div>
-    </div>
-  </div>
-<div class="container">
-
-<div class="row">
-  <div class="col-12">
-    <h2>{{ page.take_action_title }}</h2>
-  </div>
-  <div class="col-8">
-    <div class="row">
-      {% for item in page.take_action_cards.all %}
-        <a href="{{ item.link }}" class="col-6">
-          {% image item.image fill-400x200 as img %}
-          <img src="{{ img.url }}" alt="" />
-          <div>
-            {{ item.text }}
-          </div>
-        </a>
-      {% endfor %}
-    </div>
-  </div>
-  <div class="col-4">
-      <div
-        class="join-us mb-5"
-        data-cta-header="{{ cta.header | escape }}"
-        data-cta-description="{{ cta.description | escape }}"
-        data-newsletter="{{ cta.newsletter }}">
-      </div>
+    {% include "./fragments/news_you_can_use.html" %}
   </div>
 </div>
-<div class="row">
-  <div class="col">
-    <hr class="hr-emphasis mb-5">
-    <div class="row mb-5">
-      <div class="col">
-        <div class="feature-quote">
-          <h3>{% trans "We want to give users the knowledge and the power to make informed decisions by themselves to prevent abusive practices." %}</h3>
-          <p class="h6-heading">{% trans "Rishab Nithyanand, Mozilla Fellow" %}</p>
-
-
-        </div>
-        {% for orderable in page.partner_logos.all %}
-          {% image orderable.logo width-200 as logo_img %}
-          <div class="col-2">
-            {% if orderable.link %}
-              <a href="{{ orderable.link }}">
-                <img src="{{ logo_img.url }}" alt="{{ logo_img.alt }}">
-              </a>
-            {% else %}
-              <img src="{{ logo_img.url }}" alt="{{ logo_img.alt }}">
-            {% endif %}
-          </div>
-        {% endfor %}
-      </div>
-    </div>
-  </div>
 <div class="container">
+  {% include "./fragments/spotlight_posts.html" %}
+  {# @TODO Confirm these are no longer required in the template. #}
+  {% comment %} {% include "./fragments/featured_blogs.html" %} {% endcomment %}
+  {% comment %} {% include "./fragments/featured_stories.html" %} {% endcomment %}
+  {# END @TODO #}
```

Comment:
I want to highlight this: Do we need these still? The design _looks_ like we don't but I want to confirm before properly deleting it. @mmmavis

Reply:
Yeah looks like they are not needed anymore. But let's comment the template code out for now so they can act like a reminder for us to remove the rest of the related backend & CSS code.